src/components/Filter.jsx

```diff
@@ -1,17 +1,34 @@
 import React from "react";
 
-export default function Filter() {
+export default function Filter({ setOrder, setSort }) {
+  const handleOrder = (event) => {
+    event.preventDefault();
+    setOrder(event.target.value);
+  };
   const handleSort = (event) => {
     event.preventDefault();
-    console.log("heloo");
+    if (event.target.value === "default") {
+      setSort();
+    } else {
+      setSort(event.target.value);
+    }
   };
   return (
     <section className="filter-container">
       <label>
         Order
+        <select onChange={handleOrder}>
+          <option value="asc">Ascending</option>
+          <option value="desc">Descending</option>
+        </select>
+      </label>
+
+      <label>
+        Sort By
         <select onChange={handleSort}>
-          <option value="ASC">Ascending</option>
```

Comment:
When loading up the page, the order by button is set to show **ascending** however, the order on the page is showing descending order. In this case it would make sense for us to swap the order, so it shows descending as what is currently set.

Reply:
Thanks Hannah, both issues corrected now.